### PR TITLE
chore: update kn cli to 1.8.1 and func cli to 1.8.2 (#316)

### LINKED ISSUE #316: Support kn 1.8.1 and fn 1.8.1 release
fn CLI: https://github.com/knative/func/releases/tag/knative-v1.8.1
kn CLI: https://github.com/knative/client/releases/tag/knative-v1.8.1

### PR DESCRIPTION
**Pull request by lstocchi**
it resolves #316

### CODE REVIEW

File: src/cli/cli-config.json [hunk 1]
```diff
@@ -3,33 +3,33 @@
     "description": "Knative CLI tool",
     "vendor": "Red Hat, Inc.",
     "name": "kn",
-    "version": "1.7.1",
-    "versionRange": "1.7.1",
-    "versionRangeLabel": "v1.7.1",
+    "version": "1.8.1",
+    "versionRange": "1.8.1",
+    "versionRangeLabel": "v1.8.1",
     "versionLocalBuildRange": 20210101,
```
Comment (mohitsuman) [inline src/cli/cli-config.json:9]: @lstocchi Do we know why we need this field?

Comment (lstocchi) [reply src/cli/cli-config.json:9]: it was born to deal with local build but it seems it was never updated https://github.com/redhat-developer/vscode-knative/blob/main/src/cli/cli-config.ts#L93

**Comment by lstocchi** [comment]
@odockal updated the failing ui-tests. Please give them a look.

File: src/cli/cli-config.json (hunk 1 above)
Comment (mohitsuman) [reply src/cli/cli-config.json:9]: I think we can remove it and even comment out the code logic. Just one less thing to check and maintain with the version.

**Review by mohitsuman — APPROVED** [review]
LGTM

**Pull request merged**